requirements.txt

```diff
@@ -9,5 +9,6 @@ quantities==0.12.3
 redis==2.10.6
 seaborn==0.8.1
 uuid==1.30
+pytest==4.6.2
```

Comment:
we don't need this to be installed on the main application. What about this? https://pypi.org/project/requirements-test.txt/

Reply:
If we're running with tox we may also not need that

Reply:
I removed pytest from req. and I moved to install.py